examples/memeolist.query.examples.graphql

```diff
@@ -0,0 +1,62 @@
+## MEMES section
+query allMemes {
+  allMemes {
+    id
```

Comment:
Maybe should not better change the name to `getAll`.

Reply:
@camilamacedo86 The names come from here: https://github.com/aerogear/proposals/blob/master/dogfood.md#current-schema

but, they're not written on stone. @wtrocki feel free to rename everything, if you want to do it.

Reply:
Ah, one sec: I was talking about the query name defined in the schema. But, you get the idea

Reply:
@camilamacedo86 This is convention that team was using so far. This change shows as new but in fact clients are already implementing this. The most important pattern in graphql is to not change schema.